### PR TITLE
exercism.org

### PR DESCRIPTION
**Pull request by MenkeTechnologies**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 Exercism exercises in Objective-C
 
 ## Try it!
-If you've downloaded the [command-line client](http://exercism.io/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
+If you've downloaded the [command-line client](http://exercism.org/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
```
Comment (NobbZ) [inline README.md:6]: This link gives a 404, can you please find the equivalent and use that link instead?

File: README.md [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 
 ```bash
 exercism fetch objective-c
 ```
 
-In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.io/account/key).
+In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.org/account/key).
```
Comment (NobbZ) [inline README.md:12]: This link gives a 404, can you please find the equivalent replacement and use that here?

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ If you are new to open source, or are looking for an easy way in, feel free to c
 
 ### Help Add More Objective-C Exercises
 
-The easiest way to add new exercises is to translate existing exercises from other language tracks on Exercism. Here's the [full list of unimplemented exercises in Objective-C](http://exercism.io/languages/objective-c/todo) along with links to all the existing implementations.
+The easiest way to add new exercises is to translate existing exercises from other language tracks on Exercism. Here's the [full list of unimplemented exercises in Objective-C](http://exercism.org/languages/objective-c/todo) along with links to all the existing implementations.
```
Comment (NobbZ) [inline README.md:81]: This link gives a 404, can you please find the equivalent?

**Review by NobbZ — CHANGES_REQUESTED** [review]
Can you please also use HTTPS where currently HTTP is used?

File: README.md (hunk 1 above)
Comment (kytrinyx) [inline README.md:6]:
```suggestion
If you've downloaded the [command-line client](https://exercism.org/docs/using/solving-exercises/working-locally) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
```

**Review by kytrinyx — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 ## Try it!
-If you've downloaded the [command-line client](http://exercism.io/clients/cli) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
+If you've downloaded the [command-line client](https://exercism.org/docs/using/solving-exercises/working-locally) and have Objective-C installed on your machine, then go ahead and fetch the first problem.
 
 ```bash
 exercism fetch objective-c
 ```
 
-In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.io/account/key).
+In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](http://exercism.org/account/key).
```
Comment (kytrinyx) [inline README.md:12]:
```suggestion
In order to be able to submit your solution, you'll need to configure the client with your [Exercism API key](https://exercism.org/settings/api_cli).
```

**Pull request merged**